### ISSUE TITLE
Shelly TRV set to wrong value

### ISSUE THREAD

**Issue by tichachm** [efficiosoft/hass-apps#79]
Hi, I am using schedy with some zigbee thermostats for quite some time, this is working fine.
Now I added two Shelly TRV, there is one problem.

I use one Homeassistant entity to switch beteen home, away and Off mode if this modes are switched to fast (in about 10 seconds) there is a “Received value of XXX” with XXX ist the temp from previous mode event on the shelly devices, like:

`
2022-11-10 20:31:23.975342 INFO schedy_heating: --> [R:wohnzimmer] [A:climate.wohnzimmerheizung] Received value of 19.0��.
2022-11-10 20:31:24.668696 INFO schedy_heating: --> Attribute 'state' of 'input_select.heizungsmodus_schedy' changed from 'Normal' to 'Abwesend', reevaluating 5 rooms.
2022-11-10 20:31:26.046799 INFO schedy_heating: --- [R:wohnzimmer] Storing value 19.0�� before overlaying.
2022-11-10 20:31:26.055853 INFO schedy_heating: <-- [R:wohnzimmer] Value set to 16.0��.  [scheduled]
2022-11-10 20:31:26.085014 INFO schedy_heating: --- [R:balkonzimmer] Storing value 16.0�� before overlaying.
2022-11-10 20:31:26.114920 INFO schedy_heating: --- [R:alteszimmer] Storing value 16.0�� before overlaying.
2022-11-10 20:31:26.121552 INFO schedy_heating: <-- [R:alteszimmer] Value set to 15.0��.  [scheduled]
2022-11-10 20:31:26.150053 INFO schedy_heating: --- [R:toilette] Storing value 11.0�� before overlaying.
2022-11-10 20:31:26.159839 INFO schedy_heating: <-- [R:toilette] Value set to 9.0��.  [scheduled]
2022-11-10 20:31:26.187219 INFO schedy_heating: --- [R:bad] Storing value 15.0�� before overlaying.
2022-11-10 20:31:27.175106 INFO schedy_heating: --> [R:wohnzimmer] [A:climate.wohnzimmerheizung] Received value of 16.0��.
2022-11-10 20:31:27.526436 INFO schedy_heating: --> [R:toilette] [A:climate.toilette] Received value of 9.0��.
2022-11-10 20:31:29.659098 INFO schedy_heating: --> [R:alteszimmer] [A:climate.alteszimmerheizung] Received value of 15.0��.
2022-11-10 20:31:29.959206 INFO schedy_heating: --> [R:wohnzimmer] [A:climate.wohnzimmerheizung] Received value of 19.0��.
`

This only occurs on shelly trv devices (R:wohnzimmer and R:balkonzimmer), the other zigbee thermostates are working like expacted. Looks like they are to slow. Is there a way to reset the value after one minute or so for the shelly thermostates, so they do not stuck at the wrong temp?

At the moment I am running 'schedy_reevaluate' with 'mode: reset' when Homeassistant entity changed but it would be fine to reset only the shelly TRV's 1 min after every change from schedy

Thanks

**Comment by s256** [comment]
Did you ever solve this? Seeing the same here.

**Comment by tichachm** [comment]
No, I switched away from schedy and using blueprints for heating control
